x-pack/plugin/logsdb/src/javaRestTest/java/org/elasticsearch/xpack/logsdb/LogsdbRestIT.java

```diff
@@ -73,9 +74,12 @@ public void testFeatureUsageWithLogsdbIndex() throws IOException {
             List<Map<?, ?>> features = (List<Map<?, ?>>) response.get("features");
             logger.info("response's features: {}", features);
             assertThat(features, Matchers.not(Matchers.empty()));
-            Map<?, ?> feature = features.stream().filter(map -> "mappings".equals(map.get("family"))).findFirst().get();
-            assertThat(feature.get("name"), equalTo("synthetic-source"));
-            assertThat(feature.get("license_level"), equalTo("enterprise"));
+            for (var feature : features) {
```

Comment:
If for some reason there is no family field, then this assertion will not fail, while before it did.
Maybe change this assertion, so it fails if no `synthetic-source` and no `logsdb-routing-on-sort-fields` can be found?